content/docs/command-reference/move.md

```diff
@@ -164,3 +164,48 @@ $ tree
     │   └── foo
     └── data3.dvc
 ```
+
+## Example: Moving ignored files
+
+`dvc move` raises an error if it finds a file with pattern available in
+`.dvcignore` file.
+
+Let's imagine it with following scenerio:
+
+```dvc
+$ mkdir dir1 dir2
+$ echo data1 > dir1/data1
+$ echo data2 > dir2/data2
+```
+
+Create a `.dvcignore` file and insert pattern matching one of the files:
+
+```dvc
+$ echo dir1/* >> .dvcignore
+```
+
+Now lets try to move the files with `dvc move`.
+
+```dvc
+$ dvc move dir2/data2 dir3/data3
+$ dvc move dir1/data1 dir4/data4
+```
```

Comment:
Wait... You're missing a step here. `dvc add dir1` and `dvc add dir2` — but not sure you can do the latter given the .dvcignore setup. The point is `dvc move` shouldn't be used on just any file, it has to be used on DVC-tracked files or diresctories. If that's not the case please lmk and/or open a bug report in the core repo! https://github.com/iterative/dvc/issues

![image](https://user-images.githubusercontent.com/1477535/83360342-2e6aa280-a346-11ea-9df5-68e2e3fac52c.png)

Reply:
> Wait... You're missing a step here. dvc add dir1 and dvc add dir2

Ops! missed accidently.. corrected

> The other major question I see is that this new example doesn't actually show how to move ignored files!

Why would one needs to move ignored files? 
Even if someone wants to move that file (with dvc files also re-named) so they only need to again bring  that file to tracking stage and then apply `dvc move` otherwise normal `mv` will do the trick?  
And even if we wanna describe that we could add a short description of above point, no need to add an example. If you prefer.

Reply:
> missed accidently.. corrected

I don't see any changes to the PR. You haven't pushed any commits @sarthakforwet. But please read my comment closely as it has some important caveats. Thanks

Reply:
Please also note several more review comments above this review.

Reply:
Actually only https://github.com/iterative/dvc.org/pull/1378#pullrequestreview-421518565 is pending before this one. I resolved the other ones myself.

Reply:
> I don't see any changes to the PR. You haven't pushed any commits @sarthakforwet.

I did all the changes that you suggested on my local machine and decided to push when all your suggested changes are adressed.

Reply:
> Please also note several more review comments above this review.

Okay will do that..

Reply:
@sarthakforwet please don't mark this as resolved as no changes have been pushed yet. I can mark it as resolved when it is. Unresolving. Thanks

Reply:
> Why would one needs to move ignored files?

This is a good question. I don't remember what the motivation was for this new example originally but indeed, there's no point since you can't even `add` an ignored file, so you can't `move` it... 🧠 thinking